### DISCUSSION TITLE
Bakkal ve ağırlık sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#64] [Zeka Sorusu]
Bir bakkal elinde bulunan 4 ağırlık ile 1 den 40 a kadar olan tüm tamsayı ağırlıkları tartabiliyor.
Bu 4 ağırlık nelerdir?

**Comment by Berkek007** [comment]
Öncelikle 40 ı sağlaması için bu 4 sayının toplamı 40 olmalı 1 i bulmak için de 4'ünden biri kesin 1 olacak ve 2 içinse 2 + 1= 3 yapacağız ve 3 de olmalı. 4 sayıdan ilk ikisini bulduk.
1, 3, ... ....

geriye kaldı 40 - 4 = 36
1 ve 3, 5'e geldiğimizde tıkanıyor ve bize 5 + 1 +3 = 9 olduğundan 9 lazım
bu şekilde 39 dahil tüm diğer sayıları yazarız ama geriye 40 kalır yani 40 - (1 + 3 + 9) = 27

cevabımız : 1, 3, 9, 27
